File: KingdomCodeTool/wwwroot/Download/202501151754/Angular/shared/DM_BenhNhan_BHYT.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DM_BenhNhan_BHYT } from 'src/app/shared/DM_BenhNhan_BHYT.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DM_BenhNhan_BHYTService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhNhan_BHYT_Id', 'BenhNhan_Id', 'LoaiBHYT', 'SoThe', 'NgayCap', 'NgayHieuLuc', 'NgayHetHieuLuc', 'TinhThanh_CapThe_Id', 'BenhVien_KCB_Id', 'TamNgung', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id', 'Tren3Nam', 'Tren6Thang', 'NgayHuong5Nam'];
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DM_BenhNhan_BHYT";
    }
}

File: KingdomCodeTool/wwwroot/Download/202501151754/Service/Implement/DM_BenhNhan_BHYTService.cs
namespace Service.Implement
{
    public class DM_BenhNhan_BHYTService : BaseService<DM_BenhNhan_BHYT, IDM_BenhNhan_BHYTRepository>
    , IDM_BenhNhan_BHYTService
    {
    private readonly IDM_BenhNhan_BHYTRepository _DM_BenhNhan_BHYTRepository;
    public DM_BenhNhan_BHYTService(IDM_BenhNhan_BHYTRepository DM_BenhNhan_BHYTRepository) : base(DM_BenhNhan_BHYTRepository)
    {
    _DM_BenhNhan_BHYTRepository = DM_BenhNhan_BHYTRepository;
    }
    }
    }

